static/sass/custom/_search-results.scss

```diff
@@ -28,6 +32,46 @@
       padding-top: 0.75rem;
     }
   }
+
+  &__bundle-icon {
+    $padding: 0.12rem;
+    border-radius: (($padding * 2) + $icon-size) / 2;
+    border: 0.06rem solid $color-mid-light;
+    display: inline-block;
+    font-size: 0.875rem;
+    padding: $padding 0.3rem + $padding $padding $padding;
+    position: relative;
+    top: -0.12rem;
+
+    @at-root a#{&} {
+      color: $color-mid-dark;
+    }
+
+    &:hover {
+      text-decoration: none;
+    }
+
+    .search-results__icon {
+      &:first-child {
+        // Set a background to handle overlapping transparent icons.
+        background-color: #fff;
+        box-shadow: 0 0 0 0.04rem #fff;
```

Comment:
Could you use the Vanilla colour variables here please `$color-x-light`

Reply:
Done.